IOS/SwiftSeedProject/SwiftSeedProject/Interfaces/Components/NewsCollection/NewsCollectionView.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  NewsCollectionView.swift
+//  SwiftSeedProject
+//
+//  Created by Lucas Pelizza on 4/18/17.
+//  Copyright © 2017 Making Sense. All rights reserved.
+//
+
+import UIKit
+
+final class NewsCollectionView: UICollectionView, UICollectionViewDelegate, UICollectionViewDataSource {
+
+    var items: [NewsArticleViewModel] = []
+    weak var viewModelDelegate: NewsViewModel?
+    
+    func setup(viewModelDelegate: NewsViewModel)
+    {
+        self.register(UINib(nibName: NewsCollectionViewCell.identifier, bundle: nil), forCellWithReuseIdentifier: NewsCollectionViewCell.identifier)
+        self.items = []
+        self.viewModelDelegate = viewModelDelegate
+        self.dataSource = self
+    }
+    
+    func setItems(_ items: [NewsArticleViewModel]) {
+        self.items = items
+        self.reloadData()
+    }
+    
+    //MARK: - UICollectionViewDelegate
+    func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath)
+    {
+        viewModelDelegate?.selectedRow(rowVM: (items[indexPath.row]))
+    }
+    
+    //MARK: - UICollectionView Data Source
+    func numberOfSections(in collectionView: UICollectionView) -> Int
+    {
+        return 1
+    }
+    
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int
+    {
```

Comment:
Method braces and other braces (if/else/switch/while etc.) always open on the same line as the statement but close on a new line.